packages/mongo-select/src/Trigger.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import Icon from '@leafygreen-ui/icon';
+
+const orgTriggerContainer = css`
+  border-radius: 5px;
+  border: 1px solid ${uiColors.gray.light2};
+  height: 30px;
+  padding: 3px 5px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const projectTriggerContainer = css`
+  height: 45px;
+  padding: 3px 5px;
+`;
+
+const buttonContainer = css`
+  padding: unset;
+  border: none;
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 180px;
+  color: ${uiColors.gray.dark2};
+  cursor: pointer;
+
+  &::-moz-focus-inner {
+    border: 0;
+  }
+`;
+
+const selectedStyle = css`
+  margin-left: 4px;
+  font-weight: bolder;
+  flex-grow: 1;
+  text-align: left;
+`;
+
+const fontSize = css`
+  font-size: 14px;
+`;
+
+const anchorStyle = css`
+  color: ${uiColors.gray.base};
+  padding-left: 5px;
+  margin-left: 5px;
+`;
+
+const border = css`
+  border-left: 1px solid ${uiColors.gray.light2};
+`;
+
+interface TriggerProps {
+  children?: React.ReactNode;
+  selected: string;
+  className?: string;
+}
+
+export function OrganizationTrigger({
+  children,
+  selected,
+  className,
+  ...rest
+}: TriggerProps) {
+  return (
+    <div className={cx(orgTriggerContainer, className)}>
+      <button {...rest} className={buttonContainer}>
+        <Icon size="small" glyph="Building" />
+        <span className={selectedStyle}>{selected}</span>
+        <Icon size="small" glyph="CaretDown" />
+      </button>
+      <a
+        href="v2#/preferences/personalization"
```

Comment:
This may require more complexity.

For example within Cloud this is fine, however if you're outside of cloud this is going to take you relative to charts.mongodb.com or stitch.mongodb.com for example.

I'm wondering if we have this always be absolute if this would work correctly on the cloud domain as well.

Also we should have this work for non-production environments too, so people aren't redirected to a different environment.

Reply:
Was discussing this today with Dave. There are a couple of options here I guess, so wanted to know what you think makes the most sense.

Should we expose a prop, so that the consumer can let us know what environment they're working in (dev or prod)? Should we accept an object of `hrefs` that we can map to depending on product and environment? Or should we be parsing URLs?